shcomplete/repos.py

```diff
@@ -88,7 +88,7 @@ class Fish(Shell):
 @register_shell
 class Bash(Shell):
     HISTORY_FILE = re.compile(r"/.?bash_history$")
-    GITHUB_QUERY = "filename:bash_history size:>=950 -user:Dahs81"
+    GITHUB_QUERY = "filename:bash_history size:>0 -user:Dahs81"
 
```

Comment:
OK, this should go into tests. Not in this PR, but in the next one. No such hardcode is allowed in the core code.